CHANGELOG.md

```diff
@@ -38,6 +38,8 @@ Main (unreleased)
   /agent/api/v1/instances`) and list targets endpoint (`GET
   /agent/api/v1/metrics/targets`). (@marctc)
 
+- Add extra identifying labels to eventhandler integration (@hjet)
```

Comment:
Can you write down what the labels are? 

Can you also add an entry talking about supporting `extra_labels` in the config?